content/en/platform/corda/4.11/community/building-a-cordapp-index.md

```diff
@@ -21,13 +21,13 @@ title: CorDapps
 
 
 * [What is a CorDapp?](cordapp-overview.md)
```

Comment:
Add relref here?

Reply:
Thanks - just realised that wouldn't have been picked up as it contains a question mark, so might run another regex expression.

Reply:
OK done.